mesa/visualization/ModularVisualization.py

```diff
@@ -147,6 +148,7 @@ class VisualizationElement:
     local_includes = []
     js_code = ""
     render_args = {}
+    local_dir = ""
```

Comment:
Probably need a comment explaining what this default behavior is. Is it relative to the directory of user launching the Python code or something else? What if `local_dir` is precomputed on the fly to be the `__file__` of the class definition?

Reply:
Good point! Based on Tornado's source code: [[1]](https://github.com/tornadoweb/tornado/blob/7689e3f01e6ba82555f2cfac204f6f2cb7cae82b/tornado/web.py#L2610) and [[2]](https://github.com/tornadoweb/tornado/blob/7689e3f01e6ba82555f2cfac204f6f2cb7cae82b/tornado/web.py#L2751-L2766), if `local_dir` is absolute path such as `os.path.dirname(__file__) + "/../templates"`, then this path will be used.

If it's relative path such as the default `""` or `../src/templates`, I guess the current working directory is used: see [os.path.abspath](https://docs.python.org/3/library/os.path.html#os.path.abspath).

Reply:
Updated docstring.